Makefile

```diff
@@ -91,13 +102,6 @@ vet:
 generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
-go-generate: mockgen
-	go generate ./...
-
-# Build the docker image
-docker-build: test
-	docker build . -t ${IMG}
-
 # Build the image with podman
 podman-build:
 	podman build . -t ${IMG}
```

Comment:
The [`docker-*` targets](https://github.com/openshift/route-monitor-operator/blob/97404eca7e5b669d904607c6a357d459fecb9116/boilerplate/openshift/golang-osd-operator/standard.mk#L87-L95) already use a [discovered, but overrideable `CONTAINER_ENGINE` variable](https://github.com/openshift/route-monitor-operator/blob/97404eca7e5b669d904607c6a357d459fecb9116/boilerplate/openshift/golang-osd-operator/standard.mk#L18-L19), so these (`podman-build` and `podman-push`) could be removed.

Reply:
done